mdoc/Mono.Documentation/Updater/Formatters/CSharpFullMemberFormatter.cs

```diff
@@ -56,6 +56,18 @@ protected virtual string GetCSharpType (string t)
                 case "System.Void": typeToCompare = "void"; break;
                 case "System.String": typeToCompare = "string"; break;
                 case "System.Object": typeToCompare = "object"; break;
+                case "System.IntPtr":
+                    if (context != null && context.IsNativeInteger())
```

Comment:
Can we reduce these dupilcated code? And also shall we remove the context parameter here, this method looks a pure string mapping before, shall we keep it sample?